To: vim_dev@googlegroups.com
Subject: Patch 7.3.116
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.116
Problem:    'cursorline' is displayed too short when there are concealed
	    characters and 'list' is set.  (Dennis Preiser)
Solution:   Check for 'cursorline' when 'list' is set. (Christian Brabandt)
Files:	    src/screen.c


*** ../vim-7.3.115/src/screen.c	2011-02-01 18:01:06.000000000 +0100
--- src/screen.c	2011-02-09 16:59:28.000000000 +0100
***************
*** 5099,5105 ****
  #ifdef FEAT_DIFF
  		    || filler_todo > 0
  #endif
! 		    || (wp->w_p_list && lcs_eol != NUL && p_extra != at_end_str)
  		    || (n_extra != 0 && (c_extra != NUL || *p_extra != NUL)))
  		)
  	{
--- 5099,5109 ----
  #ifdef FEAT_DIFF
  		    || filler_todo > 0
  #endif
! 		    || (wp->w_p_list && lcs_eol != NUL && p_extra != at_end_str
! #ifdef FEAT_SYN_HL
! 			   && !wp->w_p_cul
! #endif
! 			)
  		    || (n_extra != 0 && (c_extra != NUL || *p_extra != NUL)))
  		)
  	{
*** ../vim-7.3.115/src/version.c	2011-02-09 16:44:45.000000000 +0100
--- src/version.c	2011-02-09 17:08:58.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     116,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
219. Your spouse has his or her lawyer deliver the divorce papers...
     via e-mail.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
